src/site/content/en/blog/economic-times-inp/index.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: Economic Times quest for fixing INP
```

Comment:
Shame no image for this post.

Reply:
Yeah, but the prior Economic Times one we did last year didn't have a hero image, so I decided to just leave it for now.